src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/Interfaces/IHomeService.cs

```diff
@@ -19,7 +19,7 @@ public interface IHomeService
 
         Dictionary<int, List<TestResultDto>> GetAllTestResults();
 
-        int GetUserPosition(string userId);
+        int[] GetUserPositions(string userId);
```

Comment:
can we create an object with Week1Place, Week2Place and so on? and rename the method to use Place, not Position?